pkg/kubelet/cm/cgroup_manager.go

```diff
@@ -0,0 +1,104 @@
+// +build linux
+
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cm
+
+import (
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+	"github.com/opencontainers/runc/libcontainer/cgroups/fs"
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+// libcontainerCgroupManager implements CgroupManager interface.
+// It Uses the Libcontainer raw fs cgroup manager for cgroup management.
+type libcontainerCgroupManager struct {
+	// Libcontainer raw fs cgroup manager.
+	fsCgroupManager cgroups.Manager
+}
+
+// Make sure that libcontainerCgroupManager implements the CgroupManager interface
+var _ CgroupManager = &libcontainerCgroupManager{}
+
+// Returns libcontainer's cgroups.config{} struct given the general cgroupConfig
+func getLibcontainerCgroupConfig(cgroupConfig *CgroupConfig) *configs.Cgroup {
+	// Extract the cgroup resource parameters
+	resourceConfig := cgroupConfig.ResourceParameters
+	resources := &configs.Resources{}
+	// skip the parameter if its a zero value
+	if resourceConfig.Memory != 0 {
+		resources.Memory = resourceConfig.Memory
+	}
+	if resourceConfig.CpuShares != 0 {
+		resources.CpuShares = resourceConfig.CpuShares
+	}
+	if resourceConfig.CpuQuota != 0 {
+		resources.CpuQuota = resourceConfig.CpuQuota
+	}
+	// Get the configs.Cgroup object
+	cgroupLibcontainer := &configs.Cgroup{
```

Comment:
Returning directly should be fine.
Also, since your are going to return immediately, the variable name could be shorter :)